lib/fastlane/actions/hipchat.rb

```diff
@@ -27,7 +27,7 @@ def self.run(params)
 
         channel = options[:channel]
         color = (options[:success] ? 'green' : 'red')
-        message = "<table><tr><td><img src=\"https://s3-eu-west-1.amazonaws.com/fastlane.tools/fastlane.png\" style=\"width:50px;height:auto\"></td><td>" + options[:message] + '</td></tr></table>'
+        message = "<table><tr><td><img src=\"https://s3-eu-west-1.amazonaws.com/fastlane.tools/fastlane.png\" width=\"50\" height=\"50\"></td><td>" + options[:message] + '</td></tr></table>'
```

Comment:
Line is too long. [190/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.